.github/ISSUE_TEMPLATE/access-request.md

```diff
@@ -39,16 +39,16 @@ Please supply all the requested information, so that your request can be process
 
 <!-- Reason why the outside collaborator needs access -->
 
-## Which MoJ team/person is responsible for this request? This must be someone with a "*.justice.gov.uk" or "*.digital.justice.gov.uk" email address.
+## Which MoJ team/person is responsible for this request? This must be a "*.justice.gov.uk" or "*.digital.justice.gov.uk" email address.
 
-This should be in the form `Awesome Team <awesome.team@digital.justice.gov.uk>` or `awesome.team@digital.justice.gov.uk`.
+This should a email address `awesome.team@digital.justice.gov.uk`
```

Comment:
Might be some works missing. Should this read "This should be an email address awesome.team@digital.justice.gov.uk"?